### DISCUSSION TITLE
Docs: Command to install Nextjs 15 RC does not work

### DISCUSSION THREAD

**Discussion by mp3por** [vercel/next.js#70718] [Help]
### What is the documentation issue?

The documentation start by giving this command `npm install next@rc react@rc react-dom@rc` to install nextjs 15 RC. 

However due to React 19 moving forward there is a peer dep error:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: f-platform-admin@1.0.0
npm ERR! Found: react@19.0.0-rc-d8c90fa4-20241001
npm ERR! node_modules/react
npm ERR!   react@"19.0.0-rc-d8c90fa4-20241001" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"19.0.0-rc-f994737d14-20240522" from next@15.0.0-rc.0
npm ERR! node_modules/next
npm ERR!   next@"15.0.0-rc.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR!
npm ERR! For a full report see:
npm ERR! /.npm/_logs/2024-10-02T11_57_09_330Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in: /.npm/_logs/2024-10-02T11_57_09_330Z-debug-0.log
````

### Is there any context that might help us understand?

I am sure you understand

### Does the docs page already exist? Please link to it.

https://nextjs.org/blog/next-15-rc

**Comment by samcx** [comment]
@mp3por Thank you for submitting an issue!

Can you try the canary instead of the release candidate? At this point, it's been several months since the 1st 15 RC was released, so it's likely behind enough on React 19 versions.

This is also technically from our blog (blogs are usually fine to become stale over time), not our documentation.

**Reply by mp3por** [reply]
`npx create-next-app@rc --empty` works as expected :)
